app/src/main/java/de/westnordost/streetcomplete/data/quest/QuestAutoSyncer.kt

```diff
@@ -55,8 +52,7 @@ import javax.inject.Singleton
     }
 
     // new location is known -> check if downloading makes sense now
-    private val locationManager = FineLocationManager(
-        context.getSystemService(Context.LOCATION_SERVICE) as LocationManager) { location ->
+    private val locationManager = FineLocationManager(context.getSystemService()!!) { location ->
```

Comment:
Even though it is not required, it would enhance readability to specify which system service is queried here

Reply:
Okay.